src/components/donationCard/DonationCard.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import ReactDom from "react-dom";
+import "../donationCard/DonationCard.scss";
+import {
+  FaHandHoldingHeart,
+  FaIdeal,
+  FaRegCalendarCheck,
+} from "react-icons/fa";
+import { GiPiggyBank } from "react-icons/gi";
+import Button from "../BtnCtA";
+import axios from "axios";
+//import { formatDistance } from "date-fns";
+
+const DonationCard = (props) => {
+  //   const createdAt = new Date(props.createdAt);
+  //   const today = new Date();
+
+  const {
+    register,
+    handleSubmit,
+    // watch,
+    // formState: { errors },
+  } = useForm();
+  const onSubmit = async (data) => {
+    console.log(data);
+    try {
+      const response = await axios.post(
+        `https://donatello-development.herokuapp.com/projects/${props.id}/donations`,
+        { donationAmount: data.amount, comment: data.comment }
+      );
+      window.location.assign(response.data.payment._links.checkout.href);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return ReactDom.createPortal(
+    <div className="popup">
+      <div className="donation-card">
+        <div className="donation-card__header">
+          <h3 className="donation-card__title">{props.titleHeader}</h3>
+          <p className="donation-card__text donation-card__text--white">
+            {props.by}
+          </p>
+        </div>
+        <form onSubmit={handleSubmit(onSubmit)} className="donation-card__main">
+          <div className="donation-card__tags">
+            {props.tags.map((tag) => {
+              return (
+                <p
+                  key={tag.name}
+                  className="donation-card__tags donation-card__tags--css"
+                >
+                  {tag.name}
+                </p>
+              );
+            })}
+          </div>
+          <p className="donation-card__text donation-card__text--black">
+            {props.description}
+          </p>
+          <h4>How much do you want to donate?</h4>
+          <div className="donation-card__amount">
+            <aside className="donation-card__amount-display donation-card__amount-display--white">
+              €25
+            </aside>
+            <aside className="donation-card__amount-inputs">
+              {[1, 5, 10, 25, 50, 100, 250].map((amount) => {
+                return (
+                  <button key={amount} className="donation-card__amount-btn">
+                    € {amount}
+                  </button>
+                );
+              })}
+              <label className="form__label--input" htmlFor="amount">
+                <input
+                  className="form__input"
+                  type="number"
+                  placeholder="Enter an amount"
+                  {...register("amount", { required: true })}
```

Comment:
We need additional validation for amount besides required (made a trello card for this)